### PR TITLE
Review

### PR DESCRIPTION
**Pull request by donice**
In this pull request, I:
- created babel, css and js directories
- Implemented the HTML file
- Created the liscence file
- Created a readMe file

### CODE REVIEW

**Comment by donice** [comment]
# Hi @donice, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights

- You added all needed files
- You used the HTML & CSS best practices
- Your github workflow was supherb
- There were no errors

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -18,7 +18,7 @@ <h1 id="title" class="text-center">freeCodeCamp Survey Form</h1>
       </header>
       <form id="survey-form">
```
Comment (donice) [inline index.html:19]: Good job using HTML5 semantic tags (e.g., header, nav) as an optional suggestion I would suggest using of `<section> <section/> `tags and include the contents inside section sections like, section with classname "main" , "aside", "footer" etc. 😊😊

File: index.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
   <body>
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
 
-    <div class="container">
```
Comment (donice) [inline index.html:12]: I would sugested the wrapping entire form tag in a `main` tag, for better semantics, and overall readabilty, and you should be good to go. 👌🏾

**Review by Alusp — APPROVED** [review]
Hi @donice , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congratulations

## Highlights

- [ ] Professional PR Summary :heavy_check_mark: 
- [ ] Use Github inline comment :heavy_check_mark: 
- [ ] Communicate Professionally :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**